src/pages/EnablePayments/IdologyQuestions.js

```diff
@@ -131,33 +131,24 @@ function IdologyQuestions({questions, walletAdditionalDetails, idNumber}) {
                     {translate('additionalDetailsStep.helpLink')}
                 </TextLink>
             </View>
-            <FormScrollView ref={formRef}>
-                <View
-                    style={styles.m5}
-                    key={currentQuestion.type}
-                >
-                    <Text style={[styles.textStrong, styles.mb5]}>{currentQuestion.prompt}</Text>
-                    <RadioButtons
-                        items={possibleAnswers}
-                        key={currentQuestionIndex}
-                        onPress={chooseAnswer}
-                    />
-                </View>
-            </FormScrollView>
-            <FixedFooter>
-                <FormAlertWithSubmitButton
-                    isAlertVisible={Boolean(errorMessage)}
-                    onSubmit={submitAnswers}
-                    onFixTheErrorsLinkPressed={() => {
-                        formRef.current.scrollTo({y: 0, animated: true});
-                    }}
-                    message={errorMessage}
-                    isLoading={walletAdditionalDetails.isLoading}
-                    buttonText={translate('common.saveAndContinue')}
-                    containerStyles={[styles.mh0, styles.mv0, styles.mb0]}
+            <FormProvider
+                formID={ONYXKEYS.WALLET_ADDITIONAL_DETAILS}
+                onSubmit={submitAnswers}
+                key={currentQuestionIndex}
```

Comment:
Here is the fix of the issue that the form doesn't re-render properly reported in https://github.com/Expensify/App/issues/32104